CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 CHANGELOG
 =========
 
+2.7.0
+-----
+
+* ignore `SessionInterface` and `UserInterface` controller action arguments
+* fixed `ExceptionListener` deprecation warning
```

Comment:
if you want to add it, there is also fixed `ControllerNameParser` deprecation warning

Reply:
:+1: